hwy/ops/x86_128-inl.h

```diff
@@ -32,7 +32,7 @@ HWY_DIAGNOSTICS_OFF(disable : 4703 6001 26494, ignored "-Wmaybe-uninitialized")
 #include <stdio.h>
 #if HWY_TARGET == HWY_SSSE3
```

Comment:
I think we should include emmintrin.h for SSE2, right?

Reply:
Yes

Reply:
Ah, my question was - shouldn't we be including emmintrin explicitly? Currently we are assuming that some other header is pulling that in for the HWY_SSE2 target. But no worries, I can make that change separately.

Reply:
> Ah, my question was - shouldn't we be including emmintrin explicitly? Currently we are assuming that some other header is pulling that in for the HWY_SSE2 target. But no worries, I can make that change separately.

There are no includes that would already have been pulling in emmintrin.h for the HWY_SSE2 target with GCC/Clang compilers, but the intrin.h include in the hwy/base.h header will automatically include emmintrin.h with the MSVC compiler on x86 if -arch:SSE2 is enabled.

The emmintrin.h include is already there in line 31 of hwy/ops/x86_128-inl.h and needs to remain to avoid compiler errors. The emmintrin.h was also already being included explicitly for the SSSE3/SSE4/AVX2/AVX3 targets prior to the addition of the SSE2 target.

Reply:
Oops, indeed. That wasn't visible in the diff and I forgot it was already there. Thanks for pointing that out :)